docs/faq.rst

```diff
@@ -0,0 +1,218 @@
+* The HIP API is less verbose than OpenCL and is familiar to CUDA developers.
+* Porting from CUDA to HIP is significantly easier than porting from CUDA to OpenCL.
+* HIP uses the best available development tools on each platform: ``amdclang++`` for AMD GPUs or ``nvcc``
+  for NVIDIA GPUs, and profilers like ``omniperf`` or ``Nsight Systems``.
+* HIP provides pointers and host-side pointer arithmetic.
+* HIP provides device-level control over memory allocation and placement.
+* HIP offers an offline compilation model.
+
+How does porting CUDA to HIP compare to porting CUDA to OpenCL?
+---------------------------------------------------------------
+
+OpenCL is fairly different from HIP and CUDA when considering the host runtime,
+but even more so when it comes to kernel code.
+The device code of HIP is a C++ dialect, while OpenCL is C99-based.
+OpenCL also does not support single-source compilation.
+
+As a result, the OpenCL syntax is a lot different from HIP, and porting tools
+have to perform complex transformations, especially when it comes to templates
+or other C++ features in kernels.
+
+Can I install both CUDA and ROCm on the same machine?
+-----------------------------------------------------
+
+Yes. Beware, that you still need a compatible GPU to actually run the compiled code.
+
+HIP detected my platform (``amd`` vs ``nvidia``) incorrectly. What should I do?
+-------------------------------------------------------------------------------
+
+If HIP is unable to detect the correct platform, or you want to manually specify
+it, set the ``HIP_PLATFORM`` environment variable to either ``amd`` or ``nvidia``.
+
+On NVIDIA platforms, can I mix HIP code with CUDA code?
+-------------------------------------------------------
+
+Yes. Most HIP data structures are typedefs to CUDA equivalents and can be
+intermixed.
+
+See the :doc:`how-to/hip_porting_guide` for more details.
+
+Compiler related questions
+==========================
+
+How to use HIP-Clang to build HIP programs?
+------------------------------------------------------
+
+``hipcc`` is just a compiler driver, meaning that it is not a compiler in itself,
+but instead calls the appropriate compilers and sets some options.
+
+The underlying compilers are ``amdclang++`` or ``nvcc``, depending on the platform,
+and can be called directly.
+
+What is HIP-Clang?
+------------------
+
+HIP-Clang is a Clang/LLVM based compiler to compile HIP programs for AMD
+platforms. It's executable is ``amdclang++``.
+
+Can I link HIP device code with host code compiled with another compiler such as gcc, icc, or clang?
+-----------------------------------------------------------------------------------------------------------
+
+Yes. HIP generates object code that conforms to the GCC ABI, and also links with libstdc++.
+This means you can compile host code with the compiler of your choice and link the
+generated host object code with device code.
+
+Can HIP applications be compiled with a C compiler?
```

Comment:
Probably we should add these here:
https://advanced-micro-devices-demo--3622.com.readthedocs.build/projects/HIP/en/3622/understand/compilers.html

Reply:
I would have to rebase on top of that. Let's wait until either is merged in and add it to the other one